loss-detection-app/docker-compose.vap.yml

```diff
@@ -2,44 +2,54 @@ version: '3.4'
 
 networks:
   loss-detection-app_edgex-network:
-    external: true   
+    external: true
 
 services:
   cv-region-of-interest:
     image: rtsf-at-checkout/cv-region-of-interest:dev
     container_name: cv-region-of-interest
     hostname: cv-region-of-interest
     environment:
+      - no_proxy=pipeline-server,mqtt
       - MQTT_DESTINATION_HOST=mqtt:1883
-      - CAMERA0_ENDPOINT=http://video-analytic:8080/pipelines/object_detection_cpu_render/1
+      # - CAMERA0_ENDPOINT=http://pipeline-server:8080/pipelines/product_detection/default
+      - CAMERA0_ENDPOINT=http://pipeline-server:8080/pipelines/product_detection/frame_store
       # Fill in camera source info below
       # for RTSP cameras
-      # - CAMERA0_SRC=rtsp://{URL}:{PORT}/{STREAM-END-POINT}
+      #- CAMERA0_SRC=file:///home/pipeline-server/video-samples/grocery-test.mp4
+      - CAMERA0_SRC=file:///home/pipeline-server/video-samples/EnterExitEvent_2second.mp4
```

Comment:
I could not find this video sample mp4 file on the local box: where could we download this mp4 video clip?

Reply:
@whbruce

Reply:
Shared file with @brian-intel